auto_invitation_to_notion/main.py

```diff
@@ -50,44 +77,49 @@ def detect_authority(user_info: Dict) -> Dict:
         # Refer: https://github.com/Lubycon/auto-invitation-to-notion/pull/1#discussion_r718537759
         page_ids = user_info[user_uid]["guest_page_ids"]
 
-        mentee_flag = LUBYCON_HUB_PAGE in page_ids
-        mentor_flag = (LUBYCON_HUB_PAGE in page_ids) and (
-            LUBYCON_MENTOR_PAGE in page_ids)
-        lubycon_flag = (LUBYCON_HUB_PAGE in page_ids) and (
-            LUBYCON_MENTOR_PAGE in page_ids) and (LUBYCON_PRIVATE_PAGE in page_ids)
+        member_flag = page_ids == [LUBYCON_SANDBOX_PAGE]
+        mate_flag = set(page_ids) == {LUBYCON_SANDBOX_PAGE, LUBYCON_MATE_PAGE}
+        admin_flag = set(page_ids) == {LUBYCON_ALL}
 
         # TODO. 동규님 피드백 (Unresolved)
         # 고정 스트링은 config file로!
         # Refer: https://github.com/Lubycon/auto-invitation-to-notion/pull/1#discussion_r718538523
-        if lubycon_flag:
-            user_info[user_uid]["authority"] = "lubycon"
+        if admin_flag:
+            user_info[user_uid]["authority"] = "admin"
```

Comment:
```suggestion
            user_info[user_uid]["authority"] = AUTHORITY[0]
```

여기는 오타가 나면 바로 버그가 나는 부분이기도 하고 추후 `authority`의 이름이 변경되었을때의 수정해야하는 스코프를 제한하기 위해서, 위에서 선언해놓은 `AUTHORITY` 배열을 활용하면 더 좋을 것 같아요. 
혹시 [Enum](https://docs.python.org/ko/3/library/enum.html)이 지원되는 환경이라면 Enum을 사용해도 좋을 것 같습니다!